### PR TITLE
 Add compute_backend_service import support

### PR DESCRIPTION
**Pull request by selmanj**
Does what it says on the tin.

/cc @danawillow

### CODE REVIEW

File: website/docs/r/compute_backend_service.html.markdown [hunk 1]
```diff
@@ -133,3 +133,11 @@ exported:
 * `fingerprint` - The fingerprint of the backend service.
 
 * `self_link` - The URI of the created resource.
+
+## Import
+
+A backend service can be imported using the `name`, e.g.
```
Comment (rileykarson) [inline website/docs/r/compute_backend_service.html.markdown]: nit: "Backend Services can..." matches [existing import blocks like `google_compute_network`](https://www.terraform.io/docs/providers/google/r/compute_network.html).

Comment (selmanj) [inline website/docs/r/compute_backend_service.html.markdown]: Done

File: website/docs/r/compute_backend_service.html.markdown [hunk 2]
```diff
@@ -133,3 +133,11 @@ exported:
 * `fingerprint` - The fingerprint of the backend service.
 
 * `self_link` - The URI of the created resource.
+
+## Import
+
+Backend services can be imported using the `name`, e.g.
+
+```
+$ terraform import google_compute_backend_service.website my_backend_service
```
Comment (rileykarson) [inline website/docs/r/compute_backend_service.html.markdown]: Nit: Can this line also be updated to import the example at the top of the page? It's a pattern I am trying to start going forward with #119. That way, readers of the Import documentation will be able to see a concrete example of how to structure an import for a resource based on an existing .tf configuration.

Comment (selmanj) [inline website/docs/r/compute_backend_service.html.markdown]: The names are pretty bad though, 'foobar', and 'blablah'. If I was reading the bottom example I might get confused as to what's what.

How about I update the example to match the import instead?

Comment (rileykarson) [inline website/docs/r/compute_backend_service.html.markdown]: Even better! :+1:

File: website/docs/r/compute_backend_service.html.markdown [hunk 3]
```diff
@@ -133,3 +133,11 @@ exported:
 * `fingerprint` - The fingerprint of the backend service.
 
 * `self_link` - The URI of the created resource.
+
+## Import
+
+Backend services can be imported using the `name`, e.g.
+
+```
+$ terraform import google_compute_backend_service.website my_backend
```
Comment (rileykarson) [inline website/docs/r/compute_backend_service.html.markdown]: s/my_backend/my-backend

File: website/docs/r/compute_backend_service.html.markdown [hunk 4]
```diff
@@ -17,31 +17,31 @@ For internal load balancing, use a [google_compute_region_backend_service](/docs
 ## Example Usage
 
 ```hcl
-resource "google_compute_backend_service" "foobar" {
-  name        = "blablah"
-  description = "Hello World 1234"
+resource "google_compute_backend_service" "website" {
```
Comment (rileykarson) [inline website/docs/r/compute_backend_service.html.markdown]: These doc changes are awesome! Looks way better than names like `"blablah"` and have `test` appear in the name of most resources.

File: website/docs/r/compute_backend_service.html.markdown (hunk 3 above)
Comment (selmanj) [inline website/docs/r/compute_backend_service.html.markdown]: good catch! fixing

File: website/docs/r/compute_backend_service.html.markdown (hunk 4 above)
Comment (selmanj) [inline website/docs/r/compute_backend_service.html.markdown]: glad to hear it!

**Pull request merged**